python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -121,7 +116,9 @@ class MakeCoaddTempExpTask(CoaddBaseTask):
 
     @copydoc \_\_init\_\_
 
-    This task has no special keyword arguments.
+    This task has one special keyword argument: passing reuse=True will cause
+    the task to skip the creation of warps that are already present in input
+    or output the repositories.
```

Comment:
Later you changed to `datasetExists(..., write=True)` so it will only search the output repos?

Reply:
Good point; I'll amend that commit to update the comment as well.